### PR TITLE
[ng] component styles: Jade to md conversion

### PR DESCRIPTION
**Pull request by chalin**
Contributes to #768.

Staged at https://angulardart-org-dev3.firebaseapp.com/angular/guide/component-styles

### CODE REVIEW

File: src/angular/guide/component-styles.md [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+</div>
+
+<?code-excerpt "lib/src/hero_details_component.css (excerpt)" region="import" title?>
+```
+  /*
+  https://github.com/dart-lang/angular2/issues/39
+  pub build fails on
+    @ import 'hero_details_box.css';
+  */
+```
```
Comment (kwalrath) [inline src/angular/guide/component-styles.md]: Why bother showing this snippet if it doesn't have real code? In fact, should we delete this section and just have a note about this not working?

Comment (chalin) [inline src/angular/guide/component-styles.md]: Kept the section (but commented out most of it) + the note about it not working yet.

File: src/angular/guide/component-styles.md [hunk 2]
```diff
@@ -262,6 +262,14 @@ You can also import CSS files into the CSS files using the standard CSS `@import
...
 on the [MDN](https://developer.mozilla.org) site.
 
+<div class="alert is-important" markdown="1">
+  {%comment%}Uncomment text below once the fix is confirmed.{%endcomment%}
+  CSS @imports cannot currently be used &mdash; see
+  [issue #39](https://github.com/dart-lang/angular2/issues/39).
+  It will be fixed in AngularDart 4.0.
+</div>
+
+{%comment%}
```
Comment (kwalrath) [inline src/angular/guide/component-styles.md]: I still see this text at https://webdev.dartlang.org/angular/guide/component-styles#!#css-imports. Looking into this... (wild guess: whitespace issue)

Comment (kwalrath) [inline src/angular/guide/component-styles.md]: I think it's a cache issue. It's fine on Safari.

**Pull request merged**